readthedocs/search/parsers.py

```diff
@@ -435,21 +465,23 @@ def _process_fjson(self, fjson_path):
                 try:
                     # Create a new html object, since the previous one could have been modified.
                     body = HTMLParser(data["body"])
-                    domain_data = self._generate_domains_data(body)
+                    # domain_data = self._generate_domains_data(body)
                 except Exception:
                     log.info("Unable to index domains.", path=fjson_path)
         else:
             log.info('Unable to index content.', path=fjson_path)
 
         return {
-            'path': path,
-            'title': title,
-            'sections': sections,
-            'domain_data': domain_data,
+            "path": path,
+            "title": title,
+            "sections": sections,
+            "domain_data": {},  # domain_data,
         }
 
     def _get_sphinx_domains(self, body):
         """
+        REMOVING THIS
+
         Get all nodes that are a sphinx domain.
 
         A Sphinx domain is a <dl> tag which contains <dt> tags with an 'id' attribute,
```

Comment:
We'll have to introduce a bit of Sphinx-specific logic (skipping `footnote` class), but maybe that's fine?

Reply:
I think we should be fine without that footnote logic, since the code will now index all of them.

Reply:
The logic will now pick up any `<dl>` containing `<dt>` elements that have an `id` attribute.